350_nationalities/100_nation_profile.md

```diff
@@ -0,0 +1,78 @@
+<div class='panel fade js-scroll-anim' data-anim='fade'>
+
+# Nation profile
+
+## `GET`{.verb} `/nationalities/` `iso_code`{.param} `/`
+
+Returns a profile featuring the combined statistics of the provided nation.
+
+The endpoint uses its own special format for [Demon](/documentation/objects#demon) and [Record](/documentation/objects#record) objects, which is a modified version of the listed form.
```

Comment:
yeeaaaaah, all post-2020 endpoints are a bit of a hack and not really meant for external use. I'd mark this one as "not stable enough for documentation", too, tbh